tutorials/nbconverted/single_cell_usage.py

```diff
@@ -0,0 +1,227 @@
+# 
+# More information about the `annotation` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.annotate)
+# 
+
+# In[22]:
+
+
+# annotating merged single-cell profile with metadata
+annotated_df = annotate(
+    profiles=sc_profile,
+    platemap=platemap_df,
+    join_on=["Metadata_well_position", "Image_Metadata_Well"],
+)
+
+# save annotated profile
+annotated_df.to_csv(anno_profile_path, compression="gzip")
+
+# displaying annotated profile
+annotated_df.head()
+
+
+# ## Noramlization Step
+# 
+# The next step is to normalize our dataset using the `normalize` function provided by `pycytominer`.
+# More information regards `pycytominer`'s `normalize` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.normalize)
+# 
+# Normalization is a critical preprocessing step that improves the quality of our dataset. It addresses two key challenges: mitigating the impact of outliers and handling variations in value scales. By normalizing the data, we ensure that our downstream analysis is not heavily influenced by these factors.
+# 
+# Additionally, normalization plays a crucial role in determining feature importance (which is crucial for our last step). By bringing all features to a comparable scale, it enables the identification of important features without biases caused by outliers or widely-scaled values.
+# 
+# To normalize our annotated single-cell morphology profile, we will utilize the normalize function from pycytominer. This function is specifically designed to handle the normalization process for cytometry data. 
+
+# In[23]:
+
+
+# normalize dataset
+normalized_df = normalize(annotated_df)
+
+# save normalized dataset 
+normalized_df.to_csv(norm_profile_path, compression="gzip")
+
+# display normalized dataset
+normalized_df.head()
+
+
+# ## Feature Selection
+# 
+
+# In the final section of our walkthrough, we will utilize the normalized dataset to extract important morphological features and generate a selected features profile.
+# 
+# To accomplish this, we will make use of the `feature_select` function provided by `pycytominer`. 
+# Using `pycytominer`'s `feature_select` function to our dataset, we can identify the most informative morphological features that contribute significantly to the variations observed in our data. These selected features will be utilized to create our feature profile.
+# 
+# For more detailed information about the `feature_select` function, its parameters, and its capabilities, please refer to the documentation available [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.feature_select).
+
+# In[24]:
+
+
+# creating selected features profile 
+features_df = feature_select(profiles=normalized_df)
+
+# saving selected features profile 
+features_df.to_csv(feat_profile_path)
```

Comment:
Please add some default arguments - it would be great to showcase some of the available options!

Also, please again use the `output()` functionality within `feature_select()` by specifying an `output_file`